src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.main {
+  display: flex;
+  justify-content: center;
+
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    width: 1024px;
```

Comment:
Great job! But please, look through the checklist again (13. [LAYOUT] - Use sass functions for dynamic max-width calculation, your catalog container should work fine for any number of cards) because I had the same mistake.

Reply:
I fixed it

Reply:
Thanks for help)